src/axom/inlet/docs/sphinx/index.rst

```diff
@@ -19,8 +19,8 @@ Introduction
 ------------
 
 Inlet provides an easy and extensible way to handle input files for simulation code.
-We provide Lua functionality but any language can be used via an inherited Reader class.
-Inlet is used to define the structure of the information expected in your input file.
+We provide JSON, Lua, and YAML functionality but any language can be used via an inherited
+Reader class. Inlet is used to define the structure of the information expected in your input file.
```

Comment:
Minor: Maybe something like "via an implementation of the Reader interface" would be a little clearer?